File: Iniciante/1017 - Gasto de Combustível/Python/1017.py
def main():
    FUEL_EFFICIENCY = 12  # 12 km per liter
    travel_time = int(input()) # in hours
    average_speed = int(input()) # km per hours
    fuel_required = calculate_fuel_required(FUEL_EFFICIENCY, travel_time, average_speed)
    print(f"{fuel_required:.3f}")


def calculate_fuel_required(FUEL_EFFICIENCY, travel_time, average_speed):
    return (travel_time * average_speed) / FUEL_EFFICIENCY


if __name__ == "__main__":
    main()

File: Iniciante/1017 - Gasto de Combustível/JavaScript/1017.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const lines = input.split("\n");
  const FUEL_EFFICIENCY = 12; // 12 km per liter
  const travelTime = Number(lines.shift()); // in hours
  const averageSpeed = Number(lines.shift()); // km per hours
  const fuelRequired = calculateFuelRequired(FUEL_EFFICIENCY, travelTime, averageSpeed);
  console.log(`${fuelRequired.toFixed(3)}`);
}

function calculateFuelRequired(fuelEfficiency, travelTime, averageSpeed) {
  return (travelTime * averageSpeed) / fuelEfficiency;
}

main();